.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: ci
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
We need to test with whatever github actions use, right?

Reply:
Yes. 14 is the oldest supported, 16 is the current LTS, and 18 is the latest available. But there are no tests here, I haven't written them yet, and the whole action is based on an existing one I found. So this CI only runs the linter.